frontEnd/src/components/App.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+
+import EmojiButton from "./EmojiButton/EmojiButton";
+import styles from "./App.module.css";
+
+function App() {
+  const handleButtonPress = label => {
+    updateSelectedMood(label);
+  };
+
+  const emojiButtonArray = [
+    { label: "🤠" },
+    { label: "🤑" },
+    { label: "🤧" },
+    { label: "🥳" },
+    { label: "😭" }
+  ];
+
+  const [selectedMood, updateSelectedMood] = useState("");
```

Comment:
Very minor but I'd prefer "null" over an empty string as the empty value but it is pretty subjective stuff.